tests/acceptance/features/apiGraph/editUser.feature

```diff
@@ -306,18 +306,37 @@ Feature: edit user
       | userRole    | role        |
       | Space Admin | Space Admin |
       | Space Admin | User        |
-      | Space Admin | User Light  |
       | Space Admin | Admin       |
       | User        | Space Admin |
       | User        | User        |
-      | User        | User Light  |
       | User        | Admin       |
       | User Light  | Space Admin |
       | User Light  | User        |
-      | User Light  | User Light  |
       | User Light  | Admin       |
```

Comment:
If the role of `Carol` is assigned to `User Light`, the file upload process fails as the `Personal` drive is not available for such a user role. Thus, the step fails. Separate steps are needed to test the same scenario for the user.